sdk/resourcemanager/Azure.ResourceManager.Core/src/Resources/ResourceIdentifier.cs

```diff
@@ -428,7 +428,7 @@ internal virtual string ToResourceString()
             StringBuilder builder = new StringBuilder(Parent.ToResourceString());
             if (IsChild)
             {
-                builder.Append($"/{ResourceType.Types.Last()}");
+                builder.Append($"/{ResourceType.Types[ResourceType.Types.Count - 1]}");
```

Comment:
To avoid `error CA1826: Do not use Enumerable methods on indexable collections. Instead use the collection directly`.